.vscode/launch.json

```diff
@@ -8,8 +8,10 @@
 			"name": "Attach to Process",
 			"type": "go",
 			"request": "attach",
-			"mode": "local",
-			"processId": "main"
+			"debugAdapter": "dlv-dap",
```

Comment:
Modified our vscode settings to work with docker.

Reply:
nice!